src/modules/TeamsList/components/Teams/components/TeamsHeader/TeamsHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import {
+  StyledTeamHeader,
+  StyledTeamHeaderRight,
+  StyledTeamHeaderSubtitle,
+  StyledTeamHeaderTitle,
+  StyledButtonBlock,
+  StyledTeamHeaderLeft,
+  HeaderManPic,
+} from './styled';
+import { Button } from '../../../../../../typography';
```

Comment:
import { Button } from 'typography';

Reply:
Alilua, this slash-points is terrible. 
done